app/src/main/scala/org/alephium/explorer/persistence/queries/BlockQueries.scala

```diff
@@ -217,10 +217,9 @@ trait BlockQueries extends TransactionQueries with StrictLogging {
     if (blocks.isEmpty) {
       DBIOAction.successful(0)
```

Comment:
Similar to above. With `QuerySplitter.splitUpdates` this check is no longer needed?